yarn.lock

```diff
@@ -18,6 +18,11 @@
     esutils "^2.0.2"
     js-tokens "^4.0.0"
 
+"@fortawesome/fontawesome-free@^5.12.0":
```

Comment:
do we need this file?
I think you added it by mistake

Reply:
No, I did not add this by mistake. We need it.